charts/stable/ring-mqtt/values.yaml

```diff
@@ -43,6 +43,20 @@ service:
         targetPort: 8554
         port: 8554
 
+probes:
+  liveness:
+    type: TCP
```

Comment:
if rtsp has such an endpoint, it will probably be HTTP aswell

Reply:
oh ok.